tests/fixtures/ptx/break_file_escape

```diff
@@ -0,0 +1 @@
+\\.+*?()|[]{}^$#&-~
```

Comment:
no need for a file for that, just create it on the fly

Reply:
Done.